docs/foundation/colors.md

```diff
@@ -3,6 +3,54 @@ sidebar_position: 2
 sidebar_label: Colors
 title: "Colors"
 ---
+## Colors
+Our default library Colors object is using **System Colors** and **Design Tokens**.
+
+### System Colors
+The System Colors are all the colors we use in our design system. (red30, grey10 and so on).
+
+### Design Tokens
+Design Tokens are contextual colors based on the system colors.  
+The design token name structure is "$[property]-[semantic]-[weight]".  (e.g $backgroundPrimaryHeavy, $textSuccessLight)
+* **Property** - The property we use this token for. The properties are:  
+  * `background`
+  * `text`
+  * `icon`
+  * `outline`
+
+
+* **Semantic** - the meaning of the color, what is the message that we want to pass using this color. The semantics are:
+  * `neutral`
+  * `primary` - the primary color of the app, means, blue for Spaces app and green for Fit app.
+  * `general`
+  * `success`
+  * `warning`
+  * `danger`
+  * `disabled`
+  * `inverted`
+  * `default`
+
+
+* **Weight** - the weight of the color (optional). The weights are:
+  * `light`
+  * `medium`
+  * `heavy`
+
+So, for example, a valid token can be: `$backgroundPrimaryHeavy` or `$textSuccess`.
+A full list of our design tokens can be found here -  
+
+### Dark Mode Support
+By using design tokens, your getting dark mode support out of the box!
+Each token is mapped to a single system color in light mode and to a (usually different) single system color in dark mode.
+For example, `$textSuccess` is mapped to `green10` in light (deafult) mode, and to `green60` in dark mode.
+All the design tokens and their mapping in light mode can be found [here](https://github.com/wix/react-native-ui-lib/blob/master/src/style/designTokens.ts), dark mode mapping can be found [here](https://github.com/wix/react-native-ui-lib/blob/master/src/style/designTokensDM.ts).
+
+### Generate Your Own Design Tokens
```

Comment:
I will add a section before this one about loading/overriding the actual tokens like we do in private using `Colors.loadSchemes({light: designTokens, dark: designTokensDM});`

Reply:
There's a section about the `loadSchemes` later on, I think I'll extend it a little bit, and I'll point to it from here

Reply:
Right. I missed it (: 
Approving anyway